### ISSUE TITLE
Mac OS says downloaded file damanged

### ISSUE THREAD

**Issue by aquav77** [schollz/streamyouraudio#6]
I am running a much older version of Mac OS. Could that be the issue?

**Comment by schollz** [comment]
Yeah, I think Go stopped supporting very old versions.

**Comment by aquav77** [comment]
I will try a  newer Mac. How reliable would this be to send live audio from computer. I assume there is a url generated for others to listen. In our case it is a church service delivered to a local AM radio station. I would hate to provide them a link and your servers changed or went down. Once they set it up from the computers it would be annoying for them to keep changing. This is a great service though. Thank You for providing!

**Comment by schollz** [comment]
Yeah it generates a URL for others to listen. I have no plans for taking the service down or changing the name. But, to be frank, the reliability really depends on sponsorship (servers/domains/bandwidth do cost me real $). If you or someone in your congregation could support me, that would go a long way to maintaining this tool and many others: https://github.com/sponsors/schollz

**Comment by aquav77** [comment]
I will see how the UI works with choosing the USB audio input and remembering it. They are using Mixlr now and it has stopped remembering on the recent update. It is fairly complicated etc. Would be great if heavier users would pay.
Are you using Icecast or shoutcast? I am not a developer and found you on a search. I guess setting up a ffmpeg server or whatever is complicated. I guess that would stream from a local computer to the internet? Would any monthly contributions use a credit card?

**Comment by schollz** [comment]
> Are you using Icecast or shoutcast? 

Neither, this simply streams from `ffmpeg` which is bundled into the binary for `streammyaudio`. No other software is needed, nothing needs to be installed. The `streammyaudio` can be just downloaded and run.

> I guess setting up a ffmpeg server or whatever is complicated. 

There is no `ffmpeg` server, its all just one program: `streammyaudio` that routes the audio (using `ffmpeg` in the background, but `streammyaudio` handles that).

> I guess that would stream from a local computer to the internet? 

Yes, that's the main purpose.

> Would any monthly contributions use a credit card?

Yes, you can choose to use a credit card or most other online banking services if you were to choose sponsorship. If you want to sponsor me, please feel free to email me (zack.scholl@gmail.com), I'd be happy to help you get you set up.

**Comment by gregsadetsky** [comment]
@aquav77 try running `xattr -c streammyaudio` (where `streammyaudio` is the binary) -- recent versions of macos will announce that a binary is "damaged" when it only needs to be "blessed" by running xattr...

**Comment by strickinato** [comment]
I recently figured out how to do the "no download" version of this on a mac os. You can running the following (if you have `ffmpeg` and `curl` installed on your machine):

```bash
ffmpeg -f avfoundation -i ":default" -f mp3 - | \
    curl -s -k -H "Transfer-Encoding: chunked" -X POST -T - \
    "https://streammyaudio.com/MYSTREAMNAME.mp3?stream=true&advertise=true"
```

Added to the instructions as well:
https://github.com/schollz/streammyaudio/pull/13

**Comment by gregsadetsky** [comment]
confirmed @strickinato , amazing! that worked immediately.
